tests/e2e-cypress/integration/3d/navigation.cy.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+import { Cartesian3 } from 'cesium'
+
+describe('Testing 3D navigation', () => {
+    context('camera limits', () => {
+        beforeEach(() => {
+            cy.goToMapView('en', {
+                '3d': true,
+            })
+        })
+        it('minimum distance from the terrain', () => {
+            cy.waitUntilCesiumTilesLoaded().then((viewer) => {
+                // Move close to the ground and try to zoom closer with the mouse wheel
+                viewer.camera.flyTo({
+                    destination: Cartesian3.fromDegrees(7.451498, 46.92805, 500),
+                    duration: 0.0,
+                })
+                cy.get('[data-cy="cesium"] .cesium-viewer').trigger('wheel', { deltaY: -5000 })
+                cy.waitUntilCesiumTilesLoaded().then(() => {
+                    expect(viewer.scene.camera.positionCartographic.height).gt(25)
+                })
+            })
+        })
+        it('maximum distance from the terrain', () => {
+            cy.waitUntilCesiumTilesLoaded().then((viewer) => {
+                // Move far from the ground and try to zoom higher with the mouse wheel
+                viewer.camera.flyTo({
+                    destination: Cartesian3.fromDegrees(7.451498, 46.92805, 4999500),
+                    duration: 0.0,
+                })
+                cy.get('[data-cy="cesium"] .cesium-viewer').trigger('wheel', { deltaY: 5000 })
+                cy.waitUntilCesiumTilesLoaded().then(() => {
+                    expect(viewer.scene.camera.positionCartographic.height).lt(5000000)
```

Comment:
same remark as above, might be a good idea to import the constant and use it here

Reply:
done